java/arcs/core/allocator/Allocator.kt

```diff
@@ -62,11 +63,20 @@ class Allocator private constructor(
     suspend fun startArcForPlan(arcName: String, plan: Plan): ArcId {
         if (plan.arcId !== null) {
             if (readPartitions(plan.arcId!!.toArcId()).isNotEmpty()) {
-                return plan.arcId!!.toArcId()
+                val arcId = plan.arcId!!.toArcId()
+                Analytics.logger.logAllocatorEvent(
+                    Analytics.Event.StartArc, arcId.name, arcId.toString())
+                return arcId
             }
         }
+
+
         val idGenerator = Id.Generator.newSession()
         val arcId = plan.arcId?.toArcId() ?: idGenerator.newArcId(arcName)
+
+        Analytics.logger.logAllocatorEvent(
+            Analytics.Event.StartArc, arcId.name, arcId.toString())
```

Comment:
Incorrect indentation/parens for wrapping.
```suggestion
        Analytics.logger.logAllocatorEvent(
            Analytics.Event.StartArc, 
            arcId.name, 
            arcId.toString()
        )
```